plugins/spu2-x/src/Windows/Spu2-X.rc

```diff
@@ -24,24 +24,6 @@ LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
 // Dialog
 //
 
-IDD_ABOUT DIALOGEX 0, 0, 291, 223
-STYLE DS_SETFONT | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "About SPU2-X"
-FONT 8, "MS Shell Dlg", 400, 0, 0x1
-BEGIN
-    DEFPUSHBUTTON   "I've Seen Enough",IDOK,91,204,111,14
-    CTEXT           "Original portions from SPU2ghz are (c) 2008 by David Quintana [gigaherz]",IDC_STATIC,9,184,273,11
-    CONTROL         114,IDC_STATIC,"Static",SS_BITMAP,45,5,203,64,WS_EX_CLIENTEDGE
-    CTEXT           "A Sound Processing Unit 2 plugin for Playstation 2 emulators.",IDC_STATIC,9,71,273,10
-    CTEXT           "For updates and news, visit the following links:",IDC_STATIC,9,98,273,10
-    CTEXT           "The SPU2-X project is derived from SPU2ghz v1.9beta released in 2008 and later modified and upgraded by the Pcsx2 Playground Team.",IDC_STATIC,25,161,242,17
-    CONTROL         "",IDC_STATIC,"Static",SS_BLACKFRAME,20,157,252,41
-    CTEXT           "Compile Date, Revision, Etc.",IDC_LABEL_VERSION_INFO,9,80,273,17
-    CTEXT           "Pcsx2 Official Git Repository @ GitHub",IDC_LINK_GOOGLECODE,9,121,273,10,SS_NOPREFIX | SS_NOTIFY
-    CTEXT           "Pcsx2 Official Website and Forums",IDC_LINK_WEBSITE,9,110,273,10,SS_NOPREFIX | SS_NOTIFY
-    CTEXT           "Brought to you by the collaborative efforts of the Pcsx2 Development Team.",IDC_STATIC,9,141,273,10
-END
-
```

Comment:
You haven't removed everything related to the about box. Look further down in the file.

Reply:
Done.